### PR TITLE
Early adoption of Marketplace pays off

### PR DESCRIPTION
**Pull request by bjgordon**

### CODE REVIEW

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+feed: true
+---
+
+# Early adoption of Marketplace pays off
+
+Local councils are starting to take advantage of the speed and ease of procurement that the Digital Marketplace brings.
+ 
+The City of Casey in Victoria published its first brief just eight days after our Beta release - an individual specialist opportunity for a developer experienced in Gravity forms. 
+ 
+In this interview, the Manager of Digital for City of Casey, Chris O'Connor, explains how happy he was to find the experience simple, clear and fast - even it its earliest Beta stage.
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: "even **in** its earliest Beta stage"

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Watch the video
+
+<div class="youtube-embed">
+  <iframe width="100%" height="400" src="https://youtu.be/c-KQqtmgm-I" frameborder="0" allowfullscreen></iframe>
+  
+  <details open data-label="content-accordion-1-example" aria-expanded="false">
+    <summary><h3>Read transcript - 2:21 min</h3></summary>
+    <div class="accordion-panel" markdown="1">
+    
+*Caption: Interview with Chris O'Connor the Manager of Digital, City of Casey - one of the first buyers in the Digital Marketplace.*
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Move the caption after the last `</div>` so that it appears after teh "Read Transcript accordion"

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ 
+Currently there are 252 registered government buyers from 78 different agencies, including the Australian Tax Office, the ABC, the Department of Health and Department of Finance, Services and Innovation to name a few.
+
+## Watch the video
+
+<div class="youtube-embed">
+  <iframe width="100%" height="400" src="https://youtu.be/c-KQqtmgm-I" frameborder="0" allowfullscreen></iframe>
+  
+  <details open data-label="content-accordion-1-example" aria-expanded="false">
+    <summary><h3>Read transcript - 2:21 min</h3></summary>
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Hmm it shows as 2:19 for me

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Chris: The simplicity of the language was a real strong point for my experience of the Marketplace - not having the RFQs the ROIs and the tenders. I'm just someone who needs a solution to my problem, and anyone who can help me connect with people who provide that solution I'm really happy with, and if it's in plain English, all the better.
+
+</div>
+  </details>
+</div>
+
+## Want to join the community? 
+
+If you work in government and have an email address ending in **.gov.au** you can create a buyer account now.
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Add the link

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+</div>
+  </details>
+</div>
+
+## Want to join the community? 
+
+If you work in government and have an email address ending in **.gov.au** you can create a buyer account now. 
+
+If your government email address is different, you can request a buyer account by emailing marketplace@digital.gov.au
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Change the email address to a link with a mailto:

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+If you want to provide digital products or services to government through the Digital Marketplace register your interest by contacting marketplace@digital.gov.au
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Change the email address to a link with a mailto:

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Early adoption of Marketplace pays off"
+category: [news]
+tag: 'digital marketplace'
+thumbnail: /images/blog-thumbnails/modernising-mygov-thumb.png
```
Comment (bjgordon) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: Is @adrian-yee going to supply an image? If not, we should use a placeholder...

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 1 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 3 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 2 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 4 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 5 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done I think!

File: _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md (hunk 6 above)
Comment (Kryan92) [inline _posts/2017-01-24-early-adoption-of-marketplace-pays-off.md]: done
